### PR TITLE
Client

### PR DESCRIPTION
**Pull request by Kyle-Chambers**
Hey guys, started the layout of client side. I'd appreciate your feedback!

### CODE REVIEW

**Review by bdduong — COMMENTED** [review]
looks good kyle!

File: client/app/components/answer.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styles from '../styling/app.css';
+
+const Answer = (props) => {
+  return (
+    <div className={styles.answerContainer}>
+
+      <p className={styles.responseFrom}>
+        Response from {props.firstName + ' ' + props.lastName + ' | ' + (props.flag === true ? '⚑' : '⚐')}
```
Comment (emsando) [inline client/app/components/answer.jsx:9]: What are those flag icons?? Is that from a library?

File: client/app/components/header.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    super(props);
+    this.state = {
+      toggle: false,
+    };
+  }
+
+  displaySubmissionForm() {
+    const boolean = this.state.toggle;
+    this.setState({
+      toggle: !boolean,
```
Comment (emsando) [inline client/app/components/header.jsx]: On an initial read this is slightly confusing, that a state property is toggle; when it's set to true does something happen automatically, like a trigger? I think the label toggle sounds like an event handler. Not a big deal though!

File: client/app/index.jsx [hunk 3]
```diff
@@ -1,13 +1,57 @@
...
+          realData: result,
+          loading: false,
+        });
+      });
+  }
+
+  render() {
+    if (this.state.loading) {
+      return (
+        <p>Page is loading</p>
```
Comment (emsando) [inline client/app/index.jsx:32]: This is really cool

File: client/app/styling/app.css [hunk 4]
```diff
@@ -0,0 +1,136 @@
+.main {
```
Comment (emsando) [inline client/app/styling/app.css:1]: Having all the css in one file takes away some of the use cases of CSS modules I think. You have to be wordier with the class names. I guess that's not such a big deal though. I'd be curious actually if there's no difference between this and separate files if the components themselves are in separate files (like they should be, and how you have them). Do you know how that works? I wonder where exactly the modularity is implemented. Just thinking out loud haha!

**Review by emsando — APPROVED** [review]

**Pull request merged**